waspc/test/UtilTest.hs

```diff
@@ -35,6 +35,11 @@ spec_toUpperFirst = do
   it "Capitalizes first letter of string" $ do
     toUpperFirst "fooBar" `shouldBe` "FooBar"
 
+spec_second :: Spec
+spec_second = do
+  it "Applies the function to the second element of a tuple of size 3." $ do
+    second negate ("1", 2, True) `shouldBe` ("1", -2, True)
```

Comment:
Keep an eye on these warnings! You need to specify which types are these, smth like `("1" :: String, 2 :: Int, True)`

Reply:
Thanks, I meant to ask what this was about, will do.